### ISSUE TITLE
After some iterations my accuracy has decrease to 0, but the loss and cross entropy are not nan.

### ISSUE THREAD

**Issue by hx173149** [tensorflow/tensorflow#4885]
Hello
I can't understand this question:
I can trained my net in a correct result in beginning but after about 4K iterations ,the accuracy suddenly decrease to zero, and the loss and cross entropy suddenly increased to a high value, it is like this curves:
train accuracy:
![image](https://cloud.githubusercontent.com/assets/5306116/19257677/2f14d1ec-8fa4-11e6-9f31-50255f2cef0e.png)
val accuracy:
![image](https://cloud.githubusercontent.com/assets/5306116/19257685/3c3b26f0-8fa4-11e6-854f-3d5b459158f6.png)
cross entropy:
![image](https://cloud.githubusercontent.com/assets/5306116/19257709/54f385de-8fa4-11e6-88c7-d5e90af88f94.png)
regular loss:
![image](https://cloud.githubusercontent.com/assets/5306116/19257716/676f5f1c-8fa4-11e6-8b87-79d92f35202d.png)

I think you can reproduce it in this link(3D Convolution, UCF101):
https://github.com/hx173149/C3D-tensorflow

thanks~

**Comment by jart** [comment]
Thanks for reaching out @hx173149. We rely on the community to provide support so this question is probably more appropriate for [Stack Overflow](http://stackoverflow.com/questions/tagged/tensorflow). We try to keep this issue tracker focused on bugs and feature requests.
